specs/flyout/flyout.md

```diff
@@ -0,0 +1,168 @@
+# Flyout
+
+## Overview
+Flyout is a focusable floating container that shows over a pages content and displays UI related to what the user is doing. Similar to a Dialog, but with positioning controls and no specific action is required, a Flyout can be used to reveal a secondary control or show more detail about an item.
+
+### Background
+A Flyout can be less intrusive to the user experience than a Dialog and should be used instead of a Dialog when information or input is needed to be displayed, but not in the center of the screen, such as in the case of collection of additional info before an action, displaying information that is only relevant some of the time, or displaying more information.
+
+### Use Cases
+- As a Dropdown for a menu, navigation bar, etc.
+- In a context menu as the secondary menu/action.
+- As a popup showing additional info for an item on the page.
+- To display a larger image of a thumbnail image in a photo gallery.
+- To display an action related to a page item
+
+### Features
+**Placement Logic:** - This component will use *Anchored Region* to be positioned relative to the Flyouts anchor element. This way it can be dynamic and adapt to the available space if `responsive` is true for the Flyout.
+
+**Light Dismiss (aka Soft Dismiss):** - A way to quickly close the Flyout by clicking outside it, or pressing the `esc` key.
+
+**Overlay:** - A control of whether or not the Flyout has an overlay effect that dims the surrounding UI when the Flyout is active.
+
+### Prior Art/Examples
+##### Context Menu
+*Note:* the "Manage, Apply, Show more options" is the Flyout for this example.
+![Context Menu](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/images/flyout-nested.png)
```

Comment:
I actually don't think this would be a flyout here - I think this would be menu/menu item which already exists. Perhaps it might be better to list these examples by Design system rather than "type". I would expect that the base flyout is primarily going to be similar to dialog or where it is a container with semantics and behaviors. It would likely be helpful to have additional examples from other design systems as currently it seems like we only have UWP examples. An example from Fluent might be their "callout" or "teaching bubble". Other design systems may call this "popover". I think having other examples will help here.